src/app/api/notification/notification.helpers.ts

```diff
@@ -0,0 +1,60 @@
+import { NotificationTaskActions } from '../core/types/tasks'
+
+/**
+ * Helper function that sets the in-product notification title and body for a given notification trigger
+ * @param actionUser The user's name that triggered this action.
+ */
+export const getInProductNotificationDetails = (actionUser: string, taskName?: string) => {
```

Comment:
missing jsdoc for para `taskName`

Reply:
Fixed in 3a9dcb86db254fe7b133359ecb68d8b0b1d765be commit